lib/Edited/prototype/listen.js

```diff
@@ -1,18 +1,44 @@
 'use strict'
-var on
-var TEXT_ALTERING_EVENT_TYPES
-
-on = require('add-event-handler')
-TEXT_ALTERING_EVENT_TYPES = require('../../text-altering-event-types')
+var on = require('add-event-handler')
+var Combokeys = require('combokeys')
+var keycode = require('keycode')
+var NON_CHARACTER_EDIT_KEYS = ['backspace', 'del', 'space', 'enter']
 
 module.exports = function () {
   var self = this
 
-  self.listener = function (e) {
-    self.discern(e)
+  // a combokeys instance for character addition
+  self.anyCharCombokeys = new Combokeys(self.element)
+  // `any-character` is a combokeys special argument that
+  // calls back on any character
+  self.anyCharCombokeys.bind('any-character', function () {
+    self.discern('character-addition')
+  })
+  self.anyCharCombokeys.stopCallback = function (e) {
+    if (e.charCode === keycode('space')) {
+      // space is handled as a different edit type
+      return true
+    }
+    return false
   }
 
-  TEXT_ALTERING_EVENT_TYPES.forEach(function (type) {
-    on(self.element, type, self.listener)
+  // a combokeys instance for the keys that do not insert characters
+  self.nonCharCombokeys = new Combokeys(self.element)
+  NON_CHARACTER_EDIT_KEYS.forEach(function (key) {
```

Comment:
Regarding the tab key and the list plugin.

The default action of the tab key [is disabled](https://github.com/wymeditor/wymeditor/blob/4db08341092d7a2778dc886a7d7514e201be4369/src/wymeditor/editor/keyboard.js#L40-L49).

The list plugin [uses combokeys](https://github.com/wymeditor/wymeditor/blob/4db08341092d7a2778dc886a7d7514e201be4369/src/wymeditor/plugins/list/jquery.wymeditor.list.js#L26-L44) for the tab action.

I expect none of this to be affected by this PR.